bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/EWKZ2Jets_ZToQQ/.log

```diff
@@ -0,0 +1,9 @@
+Starting job on  Tue Jan 18 14:07:20 CST 2022
```

Comment:
I assume the .log was committed by mistake?

Reply:
Yes. I will remove it